.gitpod.yml

```
@@ -0,0 +1,27 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Compile the protocol buffers, launch the backend with docker-compose
+    before: cd app
+    init: |
+      docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+      BASE_URL=$(gp url 3000) MEDIA_SERVER_BASE_URL=$(gp url 5000) docker-compose -f docker-compose.yml -f docker-compose.gitpod.yml up --build
+
+  - name: Install and launch the frontend with yarn
+    before: cd app/frontend
```

```suggestion
    before: cd app/web
```

And can we not rename `web` back to `frontend/blah` again... (unrelated rant)